CheckDrive.Web/CheckDrive.Web/Views/Cars/Index.cshtml

```diff
@@ -101,3 +122,97 @@
         <a rel='nofollow' href="cars/Details/${Id}">${Id}</a>
     </div>
 </script>
+
+<style>
+    .modal-lg {
+        max-width: 100%;
+        margin: auto;/* Ekranning 90% qismini egallaydi */
+    }
+
+    .modal-content {
+        width: 60%; /* Modal o'lchamini 80% qiling */
+        max-width: 700px; /* Maksimal o'lcham */
+    }
+
+</style>
```

Comment:
Is not best practise give css styles and scripts in middle of code .

Reply:
it was given away along the way